.github/workflows/buf-ci.yaml

```diff
@@ -2,15 +2,17 @@ name: Buf CI
 on:
   push:
     paths:
-      - '**.proto'
-      - '**/buf.yaml'
-      - '**/buf.lock'
-      - '**/buf.md'
-      - '**/README.md'
-      - '**/LICENSE'
-      - .github/workflows/buf-ci.yaml
+      - 'proto/**.proto'
+      - 'proto/**/buf.yaml'
+      - 'proto/**/buf.lock'
+      - 'proto/**/buf.md'
```

Comment:
```suggestion
      - 'buf.md'
```

Reply:
We don't have that file, I'd say let's just remove it?